specfile/README.md

```diff
@@ -0,0 +1,75 @@
+# Packit requirements for the spec-file library
+
+## Manipulation
+
+- We create a Python object by specifying a path of the spec-file.
+- (optional) After a change, we can reload the content (can be done by recreating of the object).
+- After the manipulation, the content is synced to the file or, we have a way to do this explicitely.
+- Be able to copy a section of spec-file to another object.
+
+## Version and release
+
+- The version and release fields can be get.
+- We can set the new version. If release is not specified, reset the release number to 1.
+
+## Changelog
+
+- We can get the content of the changelog.
+- We can add a new changelog entry:
+  - By providing only the changelog text.
+  - Optionally, field like `name` or `email` can be specified.
+
+## Sources
+
+- We can get the sources and patches.
+- We can download the sources from a lookaside cache.
+- We can download remote sources (sources in form of URL).
+- We can change the value of a source.
+- We can get archive name = basename of the first Source in SPEC file a.k.a. `Source`/`Source0`.
```

Comment:
What about the others? Why just the first one?

Reply:
I've got that from the docstring of the rebase-helper method..;) I'll remove the `first`...